R/onLoad.R

```diff
@@ -1,5 +1,7 @@
 # nocov start
 
+.Last.updated <- vector("integer", 1L) # exported variable; number of rows updated by the last := or set(), #1885
```

Comment:
This means the initial value (even if `data.table` is not loaded) is `0L`, is that preferable to `NA_integer_`?

```
library(data.table)
.Last.updated
# [1] 0
DT = data.table(a = 1)
DT[FALSE, a := 2]
.Last.updated
# [1] 0
```

Conceptually, I think `.Last.updated` is different in the two cases?